modules/vault-cluster/variables.tf

```diff
@@ -197,3 +197,37 @@ variable "force_destroy_s3_bucket" {
   description = "If 'configure_s3_backend' is enabled and you set this to true, when you run terraform destroy, this tells Terraform to delete all the objects in the S3 bucket used for backend storage. You should NOT set this to true in production or you risk losing all your data! This property is only here so automated tests of this module can clean up after themselves. Only used if 'enable_s3_backend' is set to true."
   default     = false
 }
+
+# Launch Template Extensions
+
+variable "asg_launch_mechanism" {
+  # Choices are launch_configuration and launch_template.
+  # launch_template gives you the opportunithy to declare volume encryption and
```

Comment:
You can enable volume encryption with a launch config too: https://www.terraform.io/docs/providers/aws/r/launch_configuration.html#encrypted

Reply:
Indeed it can. Comment updated.
(FWIW, that ability isn't currently exposed in this module)

Reply:
We probably should expose the ability to encrypt EBS volumes in the module for both launch configs and launch templates...